CLI/src/CLI.csproj

```diff
@@ -76,7 +76,13 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="*.cs" />
+    <Compile Include="CLI.cs" />
```

Comment:
VS try to modify csproj file, but we used to let wildcard *.cs everywhere.
For next Pull request, can you avoid to commit this kind of change, please.